realm/realm-library/src/androidTest/java/io/realm/RealmListTests.java

```diff
@@ -106,10 +109,10 @@ private void checkMethodsOnEmptyList(Realm realm, RealmList<Dog> list) {
                     case 1: list.remove(0); break;
                     case 2: list.set(0, new Dog()); break;
                     case 3: list.move(0, 0); break;
+                    default: break;
                 }
                 fail();
-            } catch (IndexOutOfBoundsException ignored) {
-            } catch (RealmException ignored) {
+            } catch (IndexOutOfBoundsException | RealmException ignored) {
```

Comment:
Java 7 construct. Even though this is in the unit tests I am not sure it is a good idea